src/uniprotkb/types/entrySection.ts

```diff
@@ -5,8 +5,11 @@ enum EntrySection {
   Interaction = 'Interaction',
   NamesAndTaxonomy = 'Names & Taxonomy',
   DiseaseAndDrugs = 'Disease & Drugs',
+  Phenotypes = 'Phenotypes',
   ProteinProcessing = 'PTM/Processing',
-  Sequence = 'Sequence & Isoforms',
+  Sequence = 'Sequence',
```

Comment:
Regarding the previous comment, I would change that enum to be functions returning a title